### ISSUE TITLE
Change Travis-CI integration to just flake8 checking

### ISSUE THREAD

**Issue by davehunt** [mozilla/Socorro-Tests#312]
We have decided to stop running our tests on Travis-CI at least in the short term, and to just use Travis-CI for basic code checking on our repositories.

In order to do this, we need to set up `.travis.yml` as follows:

```yml
language: python
python: 2.7
install: "pip install flake8"
script: "flake8 . --ignore=E501"
```

We should also remove any dependencies in `requirements.txt` that are related to running tests on Travis-CI. In this case, removing the dependency on `pytest-xdist`.

**Comment by m8ttyB** [comment]
r+ Thank you mightily @davehunt !

<img src="http://i.imgur.com/prvOkD3.gif" />

**Comment by davehunt** [comment]
It looks like this was resolved in 5e5caf5ef89b94830fae69a6d57845f827e71949 and b250f4b317da7041137f9511c67f6b377f431043. Please close issues once they're fixed. You can let GitHub do this for you by [mentioning them in the commit message](https://help.github.com/articles/closing-issues-via-commit-messages/).
